util/ujson/ujson.go

```diff
@@ -0,0 +1,269 @@
+// Package ujson implements a fast and minimal JSON parser and transformer that
+// works on unstructured json. Example use cases:
```

Comment:
Please add the source of this code and related license

Reply:
the source is from https://github.com/iOliverNguyen/ujson I already discussed this with the author